play_scraper_test.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+from play_scraper import PlayPageScraper
+
+
+def test_get_icon(directory, subdir):
```

Comment:
To me it looks like it could be easier (and cleaner) to use [tmpdir](https://docs.pytest.org/en/stable/tmpdir.html) to create both the html and the png dynamically in the test case.
In addition to removing the parameters you introduced in your conftest.py, this would also save you the problems with absolute vs. relative paths and would also make it clear right in the testcase what exactly is tested.